### PR TITLE
Feature 1 Commit

### PR DESCRIPTION
**Pull request by cormac-larkin-kainos**
First commit

### CODE REVIEW

File: feature_01.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        bank_account_number CHAR(8),
+        employee_type_id SMALLINT,
+        commission_rate DECIMAL(2,2)
+    );
+    
+    -- Add foreign key constraint
+    ALTER TABLE employee 
+    ADD CONSTRAINT fk_employee_employee_type
+    FOREIGN KEY (employee_type_id)
+    REFERENCES employee_type(employee_type_id);
```
Comment (shaunganley) [inline feature_01.sql:35]: can you add this constraint when creating the table above rather than separately?

File: feature_01.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    -- Create the employee table
+    CREATE TABLE employee (
+        employee_id SMALLINT PRIMARY KEY AUTO_INCREMENT,
+        `name` VARCHAR(255),
+        salary DECIMAL(10,2),
+        national_insurance_number CHAR(9),
+        bank_account_number CHAR(8),
+        employee_type_id SMALLINT,
+        commission_rate DECIMAL(2,2)
```
Comment (shaunganley) [inline feature_01.sql:28]: do all employees have a commission rate?

**Review by shaunganley — CHANGES_REQUESTED** [review]
comments added. can you also add some test inserts in here to show that you can create a sales employee

File: feature_01.sql (hunk 1 above)
Comment (cormac-larkin-kainos) [reply feature_01.sql]: Updated to reflect this.

File: feature_01.sql (hunk 2 above)
Comment (cormac-larkin-kainos) [reply feature_01.sql]: No, the default is NULL, commission can be added when needed.

File: feature_01.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
+DELIMITER $$
+
+    -- Use the database
+    USE kainoos_softwoorn_CormacL $$
+
+DROP PROCEDURE IF EXISTS feature_1 $$
```
Comment (shaunganley) [inline feature_01.sql:6]: is this duplicated?

**Pull request merged**